### ISSUE TITLE
Lock focus inside Modal when opened

### ISSUE THREAD

**Issue by iiroj** [iiroj/styled-modal#1]
This is an important Accessibility feature and means that tabbing or otherwise changing focused element should be constrained to the open Modal's contents. This should be by default on, but can be disable via a prop.

I'm considering using https://github.com/theKashey/focus-lock/

**Comment by iiroj** [comment]
Fixed in https://github.com/iiroj/styled-modal/pull/4
